lucien-ios/Lending/LendABookViewController.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  LendABookViewController.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/19/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 2. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Avertical_whitespace)